### PR TITLE
show relative rep with tooltip

### PR DESCRIPTION
**Pull request by dev-matan-tsuberi**
Question: `ViewProposalContainer` contains duplicate code with `ProposalContainer`. Shouldn't we just use `ProposalContainer` from `ViewProposalContainer`?

### CODE REVIEW

**Comment by dev-matan-tsuberi** [comment]
[Show relative reputation throughout the app](https://trello.com/c/nk0OCPnd/103-show-relative-reputation-throughout-the-app)

**Comment by tibetsprague** [comment]
@dev-matan-tsuberi yeah I do want to merge those as much as possible. the only thing is we dont really know what ViewProposalContainer will end up looking like, I think it could be significantly different looking. still, its true that they should probably just be the same for now.

File: src/layouts/HeaderContainer.tsx [hunk 1]
```diff
@@ -96,7 +97,11 @@ class HeaderContainer extends React.Component<IProps, null> {
                     ? <div>
                         <AccountBalance tokenSymbol={dao.tokenSymbol} balance={member.tokens} accountAddress={web3State.ethAccountAddress} />
                         &nbsp; | &nbsp;
-                        <span>{member.reputation.toFixed(2).toLocaleString()} {dao.name} reputation</span>
+                        <Tooltip placement="bottom" trigger={["hover"]} overlay={<span style={{minHeight: 'inherit !important'}}>{member.reputation} reputation</span>}>
+                          <span style={{cursor: 'pointer'}}>
```
Comment (tibetsprague) [inline src/layouts/HeaderContainer.tsx]: a pointer usually means you can click on the element and have it do something, I dont think we want that here

File: src/components/Proposal/ProposalContainer.tsx [hunk 2]
```diff
@@ -114,12 +114,18 @@ class ProposalContainer extends React.Component<IProps, null> {
         rewards.push(proposal.nativeTokenReward + " " + dao.tokenSymbol);
       }
       if (proposal.reputationChange) {
-        rewards.push(proposal.reputationChange + " reputation");
+        rewards.push(
+          <Tooltip placement="top" trigger={["hover"]} overlay={<span style={{minHeight: 'inherit !important'}}>{proposal.reputationChange} reputation</span>}>
+            <span style={{cursor: 'pointer'}}>
```
Comment (tibetsprague) [inline src/components/Proposal/ProposalContainer.tsx]: a pointer usually means you can click on the element and have it do something, I dont think we want that here

Comment (dev-matan-tsuberi) [reply src/components/Proposal/ProposalContainer.tsx]: Idk, Feels more natural to me, do you have another suggestion? Or shall I remove it?

Comment (tibetsprague) [reply src/components/Proposal/ProposalContainer.tsx]: I would just remove it. the tooltip appears immediately anyway right?

Comment (dev-matan-tsuberi) [reply src/components/Proposal/ProposalContainer.tsx]: Yes

**Comment by dev-matan-tsuberi** [comment]
Should we have a link from each proposal on "Proposals" to it's individual page?

**Comment by tibetsprague** [comment]
we do, on the view-proposal branch.

**Pull request merged**